src/ai/backend/web/server.py

```diff
@@ -187,7 +187,12 @@ async def login_handler(request: web.Request) -> web.Response:
             ),
             content_type="application/problem+json",
         )
-    creds = request["payload"]
+    raw_creds = json.loads(request["payload"])
+    try:
+        creds = json.loads(raw_creds)
```

Comment:
It seems that `json.loads()` is applied twice. Maybe `creds = json.loads(request["payload"])` should come here?

Reply:
Yes. It's my mistake. Thanks.